package.json

```diff
@@ -50,14 +51,13 @@
     "through2": "^3.0.1",
     "uglifyify": "^5.0.1",
     "vinyl": "^2.2.0",
-    "watchify": "^3.11.1",
-    "whatwg-fetch": "^3.0.0"
+    "watchify": "^3.11.1"
   },
   "devDependencies": {
+    "@types/enzyme": "^3.10.3",
```

Comment:
This is not essential, but it provides a direct way to specify the argument/return types of the `waitFor*` helpers added in this PR expect as arguments and it provides intellisense for devs who have it set up in their editors (VS Code out of the box, ale + tsserver in Vim).